prow/config/config.yaml

```diff
@@ -420,3 +482,99 @@ tide:
             from-branch-protection: true
           tiup:
             from-branch-protection: true
+          tidb:
+            branches:
+              master:
+                required-contexts:
+                  - "idc-jenkins-ci-tidb/build"
+                  - "idc-jenkins-ci-tidb/check_dev"
+                  - "idc-jenkins-ci-tidb/check_dev_2"
+                  - "idc-jenkins-ci-tidb/check_release_note"
+                  - "license/cla"
+                  - "idc-jenkins-ci-tidb/check_title"
+                  - "idc-jenkins-ci-tidb/common-test"
+                  - "idc-jenkins-ci-tidb/e2e-test"
+                  - "idc-jenkins-ci-tidb/integration-common-test"
+                  - "idc-jenkins-ci-tidb/integration-compatibility-test"
+                  - "idc-jenkins-ci-tidb/integration-copr-test"
+                  - "idc-jenkins-ci-tidb/integration-ddl-test"
+                  - "idc-jenkins-ci-tidb/mybatis-test"
+                  - "idc-jenkins-ci-tidb/sqllogic-test-1"
+                  - "idc-jenkins-ci-tidb/sqllogic-test-2"
+                  - "idc-jenkins-ci-tidb/tics-test"
+                  - "idc-jenkins-ci-tidb/unit-test"
+                skip-unknown-contexts: true
+              release-2.1:
+                required-contexts:
+                  - "idc-jenkins-ci-tidb/build"
+                  - "idc-jenkins-ci-tidb/check_dev"
+                  - "idc-jenkins-ci-tidb/check_dev_2"
+                  - "idc-jenkins-ci-tidb/check_release_note"
+                  - "license/cla"
+                  - "idc-jenkins-ci-tidb/check_title"
+                  - "idc-jenkins-ci-tidb/common-test"
+                  - "idc-jenkins-ci-tidb/e2e-test"
+                  - "idc-jenkins-ci-tidb/integration-common-test"
+                  - "idc-jenkins-ci-tidb/integration-compatibility-test"
```

Comment:
miss `integration-copr-test` and `integration-ddl-test`

Reply:
No these tests on this branch.

Reply:
<img width="1096" alt="image" src="https://user-images.githubusercontent.com/29879298/108679890-1f58ff80-7528-11eb-9088-67c5aade2b5c.png">

Reply:
See: https://github.com/pingcap/tidb/pull/22838